core/common/src/main/java/alluxio/underfs/UnderFileSystemWithLogging.java

```diff
@@ -584,15 +584,14 @@ public boolean supportsFlush() {
    * @throws IOException if an error occurs when invoking the operation on the underlying storage
    */
   private <T> T call(UfsCallable<T> callable) throws IOException {
-    T ret;
     LOG.debug("Enter: {}", callable);
     try {
-      ret = callable.call();
+      T ret = callable.call();
+      LOG.debug("Exit (OK): {}", callable);
+      return ret;
     } catch (IOException e) {
       LOG.debug("Exit (Error): {}, Error={}", callable, e.getMessage());
       throw e;
     }
```

Comment:
I'm open to suggestions on the style here. This implementation has fewer lines but the try block is larger.